src/System.Net.Security/src/System/Net/Unix/SecuritySafeHandles.cs

```diff
@@ -186,6 +186,6 @@ public override string ToString()
 
     internal abstract class SafeFreeContextBufferChannelBinding : ChannelBinding
     {
-        // TODO: To be implemented
+        // TODO (Issue #3362) To be implemented
```

Comment:
You have an awful lot of TODO 3362, but 3362 only talks about SSL_CTX_set_quiet_shutdown.